cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-admin-rights.adoc

```diff
@@ -0,0 +1,80 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-admin-rights"]
+= Tutorial: Granting admin rights
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-admin-rights
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-11-30
+
+Administration (admin) rights are not automatically granted to users that you add to your cluster. If you want to grant admin-level privileges to certain users, you will need to manually grant them to each user. You can grant admin rights from either the ROSA command line interface (CLI) or the Red Hat OpenShift Cluster Manager web user interface (UI).
+
+Red Hat offers two types of admin rights:
+
+* `cluster-admin`: `cluster-admin` rights give the admin user full privileges within the cluster.
+
+* `dedicated-admin`: `dedicated-admin` rights allow the admin user to complete most administrative tasks with certain limitations to prevent cluster damage. It is best practice to use `dedicated-admin` when elevated privileges are needed.
+
+For more information on admin privileges, see the xref:../../rosa_install_access_delete_clusters/rosa-sts-accessing-cluster.adoc#rosa-create-cluster-admins_rosa-sts-accessing-cluster[administering a cluster] documentation.
```

Comment:
This may be a comment for the topic overall, but it seems that "rights" and "privileges "are being used interchangeably. Assuming they are the same thing, wonder if we need to stick with one term so as to avoid confusion? Of course I may be overthinking this...

Reply:
I changes them all to "privileges".